sra_experiment/2_correlation_genes.ipynb

```diff
@@ -0,0 +1,1387 @@
JSvL29OXHiBC+//DIAdnZ2/PTTT1y/fp1FixZx4sQJCgsLkSSJJk2a1BpTxbw2bdpUmZ6enk52drY24UL51XbF60WLFvHJJ58QEhKCg4MDkZGR9OnTh4iICK5evUpUVBS3bt0iPDyc1157jezsbKysrHQaXtrZ2XHmzJlq3z9Av379ePvtt0lNTeXSpUtYWFhoE75wb0QiFYQG8OuvkJICH34I/ftDRAQYNLoWCA8/W1tbMjMzKS0t1UlcNjY2pKena19nZGSgUCho3rw5V69eBdC5vWNra0tERAQLFy6sdjvx8fE6rytuD+3YsYOmTZuyf/9+7X3QmmKqmJeSklLt+3BwcKixN8ETTzzB8uXLKSsrY+/evUyfPp2jR49iZmZGZGQkkZGRpKWlMXHiRNq2bYufnx83b96koKBAm0wzMzNp1aqVtszbb28ZGxsTEhLC9u3bSU5O1t4qE+6d+KoLQj0VFMA//0CvXuDrC7t3w8n721vkseXq6krLli1ZtmwZRUVFKJVKTp48yaBBg/jmm29ITU2lsLCQ6OhoQkJCqr1KBAgPD+fgwYMcPnwYjUaDUqnk6NGj2qR7u8LCQszMzLC0tCQrK4svv/zyjjEBDB8+nDVr1nDmzBkkSeLKlSukp6fj6uqKubk5q1evpqSkBI1GQ1JSEgkJCUD5/cqcnBwMDAy0V74GBgb88ccf/P3332g0GiwsLFAoFBgYGGBra4ubmxvLly9HqVRy/vx5Nm/eTHh4eK37MyIigi1btnDgwAGRSOtBXJEKQj0dOgQaDXTvDh07llfzXroEnp76jqzhPCz9POVyOTExMSxcuJA+ffoAEBYWxrx588jKymL06NEolUr8/Px48803ayzH1taWlStXsnTpUmbOnImBgQGurq6888471S4fGRnJnDlz8PDwoE2bNkRERLB27dpaY3J3dyckJIS8vDxmzpyprX7+4IMPsLe3JyYmhvfff59+/fqhUqlo27YtM2bMAODw4cMsWbKEkpIS7OzsiI6OxsTEhOvXr/P222+TlZWFmZkZoaGh2gS4fPly3n77bfz9/WnSpAnTpk3D19e31v3p7u6OgYEBnTt31qkaF+6NTKroJ/KYGzp0KLGxsfoOQ2iEpk2DVasgOhoMDeGDD0Amg1mzqi47ceKDj+9eJSYm4uLiou8whAdk7NixhIWFMWLEiFqXq+lzIc6dompXEOptzx5wdi5PogBt2kBqKpSV6TcuQbiThIQEzp07R0hIiL5DadREIhWEerh0CS5cgKee+t80JydQKqGG222C8FCYM2cO48ePr3aYVeHeiHukglAPe/aU//3vA4mA8kQK5a147ewefEyCcDfef/99fYfwyBBXpIJQD3v2lFflVuplQOvWYGQEV67oLy5BEB4ckUgFoR5+/x369i1vXFTBwAAcHUUiFYTHhUikglBHubmQna1brVvByUk0OBKEx4VIpIJQR0lJ5X87dqw6z8kJVCrR4EgQHgcikQpCHVU8O8HZueq8iuFVRfWuIDz6RCIVhDpKSgK5HNq2rTqvdWswNhaJ9EFZsWIFr7/+ep3X//bbb/H19cXNzY3c3FxOnjxJcHAwbm5u7N+/nzFjxrBp06Y6lV2fdYXGQXR/EYQ6+vtvaNeuvIXu7QwMyru+/PcxuY3eyfs8eLC7u/6GIFSr1SxZsoSNGzfSqVMnAD755BOef/557UPAv/nmG73FJzz8xBWpINRRUlL190crtGgBlZ7iJTykbty4gVKp5MlKz7/LyMigQ4cOeoxKaExEIhWEOigrKx/RqLr7oxWaN4ecHNFyt6GtXr0af39/3Nzc6N+/P0eOHAHKryxnz56Nm5sbAwcO5PTp09p1nJ2duVKpnn3u3LlER0dz6dIlBgwYAICnpydjx44lMDCQ1NRUJk+ejJubGyqVqkoMmzdvJiQkBE9PT1566SWdR7j99ttvDBgwAHd3dxYsWIAYzvzRJxKpINRBaioUF9/5irSsrLybjNAwkpOTWb9+PZs3byY+Pp6vvvpK+9SSAwcOMHDgQE6cOEHfvn3597//fcfy2rZty48//gjA8ePHWbduHfv378fOzo6YmBji4+Mxuq3ufv/+/axatYpPP/2UI0eO4O7uzsyZMwHIyckhMjKSGTNm8Mcff9CmTRv+/PPPBt4LwsNGJFJBqIOKri+1XZG2aFH+V1TvNhy5XI5KpeLixYuo1WocHBxo898m0u7u7gQEBCCXy4mIiOD8+fP3JYYNGzYwceJE2rdvj0KhYPLkySQmJpKens4vv/xChw4dGDBgAIaGhrzwwgu0qPggCI8skUgFoQ4qur7UdkXavHn53+vX7388jwsnJyfmzZvHihUr8PX15bXXXiMrKwtAJ2GZmJigVCopLS1t8BgyMjJYvHgxHh4eeHh44OXlhSRJZGVlkZ2dTevWrbXLymQybG1tGzwG4eEiWu0KQh0kJYGFBdR2jmzWrHzoQJFIG1ZYWBhhYWEUFBTw1ltv8eGHH2qvSmtiampKcXGx9vW1a9doVXmA5Htga2vL5MmTCQ8PrzLvypUrXK00CockSWRmZtZpO0LjIa5IBaEO/v67/Gq08hi7t1MooGlTUbXbkJKTkzly5AgqlQojIyOMjY0xMLjzaaxTp078+OOPaDQafvnlF44fP17nGEaNGsXq1au5cOECAPn5+ezatQuAgIAALly4wN69eyktLWXdunVcF7+kHnniilQQ6iApCXr0uPNyLVo8Glek+uznWZlKpWLZsmVcvHgRQ0ND3NzcWLBgARs3bqx1vfnz5zN37lzWr19PYGAggYGBdY4hKCiIwsJCoqKiSE9Px9LSEl9fX0JCQmjWrBkff/wxixYt4o033iAiIoLu3bvXeVtC4yCTRNtsAIYOHUpsbKy+wxAageJiMDeHt98u/wewenX1y379dfnV65Il5a8nTnwwMdZHYmIiLi4u+g5DeMjU9LkQ505RtSsI9+ziRZCk2hsaVWjRAvLyQK2+/3
```

Comment:
I think this should be something like "What are the accessory genes with high correlation?" (not sure how to do suggested changes on Jupyter notebooks)